bundles/binding/org.openhab.binding.zibase/src/main/java/org/openhab/binding/zibase/internal/ZibaseGenericBindingProvider.java

```diff
@@ -49,6 +49,11 @@
     static final HashMap<String, Vector<String>> itemIdNameMap = new HashMap<String, Vector<String>>();
 
     /**
+     * Map that allows to get registered item by name
+     */
+    static final HashMap<String, Item> itemMap = new HashMap<String, Item>();
```

Comment:
I just had another look at this item caching... in fact I'm using it in ZibaseListener.publishEvents(). I just need to get the item so I can ensure that the postupdate() is synchronized for a given item. 

Do you think it's useless or that there is another way to do this ?